frappe/public/js/frappe/list/list_view.js

```diff
@@ -1474,7 +1474,7 @@ frappe.views.ListView = class ListView extends frappe.views.BaseList {
 				this.data = this.data.filter((d) => !names.includes(d.name));
 				for (let name of names) {
 					this.$result
-						.find(`.list-row-checkbox[data-name='${name}']`)
+						.find(`.list-row-checkbox[data-name='${name.replace(/'/g, "\\'")}']`)
```

Comment:
Use some escape function instead.

Reply:
I tried some, but this was the only thing that worked